packages/page-pattern-modal/src/styles/page-pattern-modal.scss

```diff
@@ -54,14 +66,20 @@ $sidebar-width-desktop: 324px;
 		overflow-y: scroll;
 		-webkit-overflow-scrolling: touch;
 		// prevents a 1px strip of content being visible after scrolling
-		margin-top: -1px;
+		margin-top: 60px;
```

Comment:
If we are changing this margin that comment should probably be removed.

Reply:
yup good catch